src/Migrations.php

```diff
@@ -109,7 +113,9 @@ public static function generate(array $options)
         $migrationPath = rtrim($migrationsDir, '\\/') . DIRECTORY_SEPARATOR . $versionItem->getVersion();
         if (!file_exists($migrationPath)) {
             if (is_writable(dirname($migrationPath)) && !$optionStack->getOption('verbose')) {
-                mkdir($migrationPath);
+                if (!mkdir($migrationPath) && !is_dir($migrationPath)) {
```

Comment:
`is_dir()` check is redundant, as it already checks above with `file_exists()` + `is_writable()`.